driver-core/src/main/java/com/datastax/driver/core/RequestHandler.java

```diff
@@ -333,8 +329,8 @@ private void setFinalException(
 
     cancelPendingExecutions(execution);
 
-    span.setStatus(StatusCode.ERROR, exception.toString());
-    span.end();
+    tracingInfo.setStatus(TracingInfo.StatusCode.ERROR, exception.toString());
```

Comment:
Is this exception recorded before or the only information about it is the status message?

Reply:
I encourage you to check it.

Reply:
Haven't found any proof that this exception is recorded so I think we should call `tracingInfo.recordException(e)` like in `setFinalResult`

Reply:
Done.